src/main/resources/templates/config/open-poker-config.vm

```diff
@@ -0,0 +1,35 @@
+$webResourceManager.requireResource("com.aprey.jira.plugin.openpoker.open-poker:open-poker-config-resources")
+<html>
+<head>
+    <title>Open poker configuration</title>
+</head>
+<body>
+
+<header class="aui-page-header">
+    <div class="aui-page-header-inner">
+        <div class="aui-page-header-main">
+            <h1>Open poker configuration</h1>
+        </div>
+    </div>
+</header>
+<h2>Permissions</h2>
+<form action="$action.getURL()" class="aui" method="POST">
+    <input type="hidden" id="allowedProjects" name="allowedProjects" value="$action.getAllowedProjectsValue()"/>
+    <fieldset>
+        <label for="allowedProjects" class="paramName">Projects:</label>
+        <select id="allowedProjectsSelect" name="allowedProjectsSelect" style="padding: 5px 0 0 0" multiple>
```

Comment:
I think it might be good to limit the ``width`` of the ``select``, because it becomes quite long on big screens.
I'd suggest: ``max-width: 800px``.